tests/testthat/_snaps/install.md

```diff
@@ -3,6 +3,12 @@
     Code
       install("bread@0.1.0")
     Output
+      Downloading 1 package(s) and their dependencies
+      Done
+      
+      The following package(s) will be installed into the library
```

Comment:
Drop "into the library" ?

Reply:
Yeah, let's drop it. Maybe:

```
The following packages will be installed:

- abc
- xyz

Packages will be installed into ~/path/to/library.
```

Reporting the library path might be useful just to confirm the library path is what the user expects?

Reply:
Or maybe we just always print this and remove the tests-running conditional: https://github.com/rstudio/renv/blob/6ad88516825f92f6d0fd919dfd8395a1d07462ec/R/install.R#L197-L205

Reply:
Moved that into the preamble, and removed tweaked other test specific code.